library/libraries.js

```diff
@@ -738,7 +738,7 @@ var d41d8cd98f00b204e9800998ecf8427e_LibraryDetectorTests = {
 
     'Move': {
        icon: 'move',
-       url: 'http://movelang.org/',
+       url: 'https://visionmedia.github.io/move.js/',
```

Comment:
This is the wrong URL for this library.  Unfortunately, the original domain no longer contains information about it.  See this for more details: [Move — a programming language](https://rsms.me/move-programming-language)

If you'd like, you can revise this entry to use new tests for the current Move library URL.